src/irr/asset/CPLYMeshFileLoader.cpp

```diff
@@ -70,8 +66,7 @@ asset::SAssetBundle CPLYMeshFileLoader::loadAsset(io::IReadFile* _file, const as
         return {};
 
     SContext ctx;
-
-	ctx.File = std::move(std::make_unique<io::IReadFile>(_file));
+	ctx.File = std::move(core::make_smart_refctd_ptr<io::IReadFile>(_file));
```

Comment:
use raw pointer and don't grab, you won't be using the file anymore after `loadAsset` returns.